static/lang/re-en.json

```diff
@@ -2329,6 +2329,12 @@
                     "Note": "When you Sense Direction, you get a result one degree of success better than you rolled."
                 }
             },
+            "RelicGift": {
+                "DivineRetribution": {
+                    "CriticalSuccess": "If the target is evil, it is also @UUID[Compendium.pf2e.conditionitems.Item.MIRkyAjyBeXivMa7]{Enfeebled 1} until the start of your next turn on a critical hit.",
```

Comment:
I believe the critical success and the regular success have their enfeebled values swapped.